ui_transcendent_patch.py

```
@@ -0,0 +1,72 @@
+import sys
+
+def patch_panel():
+    path = 'src/scripts/components/panel.js'
+    with open(path, 'r') as f: lines = f.readlines()
```

Ruff (0.15.2)

[warning] Unnecessary mode argument

Remove mode argument

(UP015)